fedbiomed/common/config.py

```diff
@@ -73,6 +83,38 @@ def _COMPONENT_TYPE(cls):  # pylint: disable=C0103
     def _CONFIG_VERSION(cls):  # pylint: disable=C0103
         """Abstract attribute to oblige defining component type"""
 
+    def load(
+        self,
+        name: str,
+        root: str | None = None,
+        auto_generate: bool = True
+    ) -> None:
+        """Load configuration from given name and root
+
+        This implementation allows to load configuration after Config class
+        is instantiated.
+
+        Args:
+            name: Name of the config file
+            root: Root directory where component files will be saved
+            auto_generate: Generated all component files, folder, including
+                configuration file.
+        """
+        self.name = name
+
+        if root:
+            self.path = os.path.join(root, CONFIG_FOLDER_NAME, self.name)
+            self.root = root
+        else:
+            self.path = os.path.join(CONFIG_DIR, self.name)
+            self.root = ROOT_DIR
+
+        if auto_generate:
```

Comment:
Shouldn't we read the config in the case where `auto_generate` is False but the config already exists ?

```suggestion
        if auto_generate:
            self.generate()
        elif self.is_config_existing():
            self.read()
```

Reply:
Hi @mvesin,

Yes I makes sense it add this option as well thanks.

Reply:
Pushed this fix, as we finally forgot it